### PR TITLE
fix: #436 fix styles in storybook, refactor select, create ThemeSwitcher, add ThemeSwitcher to storybook components

### PR DESCRIPTION
**Pull request by AKiroi**
- fixed styles in storybook;
- refactor Select component;
- create ThemeSwitcher component;
- add ThemSwitcher to storybook components;
image

### CODE REVIEW

File: mw-webapp/.storybook/preview.ts [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import type { Preview } from "@storybook/react";
+import { Theme, ThemeWorker } from './../src/utils/ThemeWorker';
+
+ThemeWorker.setTheme(Theme.LIGHT);
```
Comment (tritonJS826) [inline mw-webapp/.storybook/preview.ts:4]: Does it possible to add theme switcher somewhere in the ui, to get ability for user change components styles?

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/.storybook/preview.ts (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/.storybook/preview.ts:4]: Yes, it's possible, I can add ThemeSwitcher to storybook component like this, 
image
but we have problem with Defaut doc(autodocs) with switcher position .
image

Comment (AKiroi) [reply mw-webapp/.storybook/preview.ts:4]: we can turn off autodocs, the same options are available in the storybook component
image

Comment (AKiroi) [reply mw-webapp/.storybook/preview.ts:4]: It's possible to add ThemeSwitcher to storybook component without docs page. Add this funcionality?
image
image

File: mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx
Comment (tritonJS826) [inline mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx:1]: File name with component should start with capital letter

Comment (tritonJS826) [inline mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx:1]: You will replace themeSwitcher based on select on switcher based on moon and sun icons in the next PR, right?

File: mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  return (
+    <Select
+      label="Theme: "
+      value={currentTheme}
+      name="theme"
+      options={themeOptions}
+      onChange={(value) => {
+        ThemeWorker.setTheme(value);
+      }}
```
Comment (tritonJS826) [inline mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx:24]: Dont create unnecessary wrappers
```suggestion
      onChange={ThemeWorker.setTheme}
```

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx (hunk 2 above)
Comment (AKiroi) [reply mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx:24]: fixed

File: mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx
Comment (AKiroi) [reply mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx:1]: Yes, create a new task in the Todo list of project. I'll do it tomorrow

Comment (AKiroi) [reply mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx:1]: File name with component starts with capital letter in ThemeSwitcher component. Maybe you mean folder  themeSwitcher/ThemeSwitcher.tsx?

Comment (tritonJS826) [reply mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.tsx:1]: hm, no it's ok)

**Pull request merged**